backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/nodes/AbstractDefinitionNodeDescriptionProvider.java

```diff
@@ -78,6 +78,16 @@ public AbstractDefinitionNodeDescriptionProvider(EClass eClass, IColorProvider c
      */
     protected abstract Set<NodeDescription> getReusedChildren(IViewDiagramElementFinder cache);
 
+    /**
+     * Implementers should provide the set of {@link NodeDescription} that is reused as border node in that Usage node
+     * description.
+     *
+     * @param cache
+     *            the cache used to retrieve node descriptions.
+     * @return the set of {@link NodeDescription} that are added as reused child.
```

Comment:
that are added as reused child. => that are reused as border nodes.

Reply:
Done